src/OpenTelemetry/Metrics/Processors/MetricConsoleExporter.cs

```diff
@@ -34,12 +34,45 @@ public override void OnEnd(MetricItem data)
             {
                 var tags = metric.Attributes.ToArray().Select(k => $"{k.Key}={k.Value?.ToString()}");
 
+                string valueDisplay = string.Empty;
+                if (metric is ISumMetric sumMetric)
+                {
+                    if (sumMetric.Sum is double doubleSum)
```

Comment:
Do we need these if-else conditions or we simply call ToString for whatever point types?

Reply:
just to demonstrate that the sum is available as double, if the exporter's backend wants it as double.
(yes in this console, i could just ToString() it.)